source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
+
+
+Avaliable timeouts
+------------------
+
+pfSense's captive portal has 3 integrated timeout options : 
+
+- **"Idle timeout"** causes users to be automatically disconnected after an amount of time of network inactivity.
```

Comment:
This list should be a field list, and you can remove the quotes as well.

Example:

```
:Idle Timeout: Users will be ...
:Hard Timeout: Users will be ...
```

https://docs.netgate.com/pfsense/en/latest/references/formatting-guide.html#lists